documentation/missionmaker/modules.md

```diff
@@ -70,217 +75,449 @@ If you are worried that players haven't updated ACE or other mods to the version
+`Default value: 1`
+7. **AI Damage (Number)**<br>
+What is the damage an AI can take before being killed?<br>
+`Default value: 1`
+8. **AI Unconsciousness (Option)**<br>
+Allow AI to go unconscious.<br>
+`Default value: "50/50"`
+9. **Prevent instant death (Boolean)**<br>
+Have a unit move to unconscious instead of death.<br>
+`Default value: No`
+10. **Bleeding coefficient (Number)**<br>
+Coefficient to modify the bleeding speed.<br>
+`Default value: 1`
+11. **Pain coefficient (Number)**<br>
+Coefficient to modify the pain intensity.<br>
+`Default value: 1`
+12. **Pain coefficient (Boolean)**<br>
+Keep unit status synced. Recommended on.<br>
+`Default value: Yes`
+
+
+### 2.2 Advanced Medical Settings
+
+This module allows you to change the default Advanced Medical Settings, when [2.2 Medical Settings](#2-2MedicalSettings) is set to "Advanced".
+
+**Settings:**
+
+1. **Enabled for (Option)**<br>
+Select what units the advanced medical system will be enabled for.<br>
+`Default value: "Players only"`
+2. **Enable Advanced wounds (Boolean)**<br>
+Allow reopening of bandaged wounds?<br>
+`Default value: No`
+3. **Vehicle Crashes (Boolean)**<br>
+Do units take damage from a vehicle crash?<br>
+`Default value: Yes`
+4. **Allow PAK (Option)**<br>
+Who can use the PAK for full heal?<br>
+`Default value: "Medics only"`
+5. **Remove PAK on use (Boolean)**<br>
+Should PAK be removed on usage?<br>
+`Default value: Yes`
+6. **Locations PAK (Option)**<br>
+Where can the personal aid kit be used?<br>
+`Default value: "Vehicles & facility"`
+7. **Allow Surgical kit (Option)**<br>
+Who can use the surgical kit?<br>
+`Default value: "Medics only"`
+8. **Remove Surgical kit (Boolean)**<br>
+Should Surgical kit be removed on usage?<br>
+`Default value: Yes`
+9. **Locations Surgical kit (Option)**<br>
+Where can the Surgical kit be used?<br>
+`Default value: "Vehicles & facility"`
+
+
+### 2.3 Revive Settings
+
+This modules allows a mission maker to limit the amount of revives for units in basic and advanced medical. 
+
+**Settings:**
+
+1. **Enable Revive (Option)**<br>
+Enable a basic revive system<br>
+`Default value: "disable"`
+2. **Max Revive time (Number)**<br>
+Max amount of seconds a unit can spend in revive state<br>
+`Default value: 120`
+3. **Max Revive lives (Number)**<br>
+Max amount of lives a unit. 0 or -1 is disabled.<br>
+`Default value: -1`
+
+
+### 2.4 Set Medic Class
+
+Using this module you can define which unit class is defined as a medic / doctor. 
+
+**Settings:**
+
+1. **List (String)**<br>
+List of unit names that will be classified as medic, separated by commas.<br>
+`Default value: ""`
+2. **Is Medic (Boolean)**<br>
+Medics allow for more advanced treatment in case of Advanced Medic roles enabled<br>
+`Default value: "Regular medic"`
+
+
+### 2.5 Set Medical Facility
+
+Defines an object as a medical facility for CMS. This allows for more advanced treatments. Can be synced with buildings and vehicles.
```

Comment:
>for CMS
This could be removed. It is no longer called CMS (and works for both basic and advanced).